_includes/scenarios/integrate/requests-over-templates.md

```diff
@@ -120,9 +131,22 @@ If you want to accept the Request you can do so by calling the `PUT /api/v2/Requ
 }
 ```
 
-In the response you can see the Request has moved to status `Decided`. This is where the enmeshed Runtime steps in and handles the Request based on you decision. Because you accepted the Request, the enmeshed Runtime will send your Response to the Templator by creating a Relationship. This behavior can be observed by querying the Request again after a few seconds (`GET /api/v2/Requests/Incoming/{id}`). When the Request is in status `Completed` you can query the created Relationship (`GET /api/v2/Relationships`, query parameter `template.id=<id-of-the-template>`).
+In the response you can see the Request has moved to `status` `"Decided"`.
+This behavior can be observed by [querying the Request]({% link _docs_use-cases/use-case-consumption-get-incoming-request.md %}) again after a few seconds.
+Now, the enmeshed Runtime steps in and handles the Request based on you decision.
+The Response is created with the appropriate ResponseItems.
+
+If there is already an active Relationship between the Connectors, the Response will be sent back to the Sender via [Message]({% link _docs_integrate/data-model-overview.md %}#message) and the Request will move to `status` `"Completed"`.
+The Sender, then, can fetch it by [synchronizing updates of the Backbone]({% link _docs_use-cases/use-case-transport-synchronize-updates-of-backbone.md %}).
+
+However, if there is no active Relationship between the Connectors, yet, a Relationship will be created, which has the `status` `"Pending"` for now.
+It contains a [RelationshipChangeRequest]({% link _docs_integrate/data-model-overview.md %}#relationshipchangerequest) with a `content` of type [RelationshipCreationChangeRequestContent]({% link _docs_integrate/data-model-overview.md %}#relationshipcreationchangerequestcontent), that in turn contains the Response to the Request.
+This Relationship will be sent back to the Sender via Message.
+Then, the Request will be set to `status` `"Completed"` and you can [query the Relationship]({% link _docs_use-cases/use-case-transport-query-relationships.md %}) using the query parameter `template.id=<id-of-the-template>`.
+Until the RelationshipChangeRequest is answered, no new Request will be created by [loading the template](#load-the-relationshiptemplate-and-get-the-request).
```

Comment:
We should write "RelationshipTemplate" here instead of "template".

The same applies to another location and to the title of the scenario. I'd rename the scenarios to "Requests via RelationshipTemplates" and "Requests via Messages".